newIDE/app/src/Profile/Subscription/SubscriptionDetails.js

```diff
@@ -25,59 +39,83 @@ const SubscriptionDetails = ({
+        <Text size="block-title">My online services subscription</Text>
       </Line>
-      {subscription.planId ? (
+      {userPlan && userPlan.planId ? (
         <>
-          <Line>
-            <Text>
-              <Trans>
-                You are subscribed to <b>{subscription.planId}</b>.
-                Congratulations! You have access to more cloud projects,
-                leaderboards, player feedbacks, cloud builds.
-              </Trans>
-            </Text>
-          </Line>
-          <ResponsiveLineStackLayout justifyContent="flex-end">
-            <LeftLoader isLoading={isManageSubscriptionLoading}>
-              <FlatButton
-                label={<Trans>Manage online</Trans>}
+          <PlanCard
+            plan={userPlan}
+            actions={[
+              <LeftLoader
+                key="manage-online"
+                isLoading={isManageSubscriptionLoading}
+              >
+                <FlatButton
+                  label={<Trans>Manage online</Trans>}
+                  primary
+                  onClick={onManageSubscription}
+                />
+              </LeftLoader>,
+              <RaisedButton
+                key="manage"
+                label={<Trans>Manage subscription</Trans>}
                 primary
-                onClick={onManageSubscription}
-              />
-            </LeftLoader>
-            <RaisedButton
-              label={<Trans>Upgrade or cancel</Trans>}
-              primary
-              onClick={() =>
-                openSubscriptionDialog({ reason: 'Consult profile' })
-              }
-            />
-          </ResponsiveLineStackLayout>
+                onClick={() =>
+                  openSubscriptionDialog({ reason: 'Consult profile' })
+                }
+              />,
+            ]}
+            isHighlighted={false}
+          />
         </>
       ) : (
         <>
-          <Line>
-            <Text>
-              <Trans>
-                Get a subscription to unlock more cloud projects, more
-                leaderboards and more player feedbacks. You'll also have more
-                cloud builds to publish your game to Android and on desktop.
-              </Trans>
-            </Text>
-          </Line>
-          <Line justifyContent="flex-end">
-            <RaisedButton
-              label={<Trans>Choose a subscription</Trans>}
-              primary
-              onClick={() =>
-                openSubscriptionDialog({ reason: 'Consult profile' })
-              }
-            />
-          </Line>
+          <Paper background="medium" variant="outlined">
+            <LineStackLayout alignItems="center">
+              <img src="res/diamond.svg" style={styles.diamondIcon} alt="" />
+              <Column expand>
+                <Line>
+                  <Column noMargin>
+                    <Text noMargin>
+                      <Trans>
+                        Unlock more one-click exports and other exclusive
```

Comment:
Let's take the opportunity to change this text.
We should mention leaderboards, cloud projects and exports

Reply:
"Unlock more one-click exports, cloud projects, leaderboards and remove the GDevelop splashscreen."